packages/client/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,298 @@
+              justifyContent: "center",
+              alignItems: "center",
+              marginBottom: "100px",
+            }}
+          >
+            <img src="../../../public/images/towerTwoBase-1.png" alt="Tower" />
+            <Typography
+              variant="h5"
+              sx={{
+                margin: "0 80px",
+                color: "#104603",
+                fontFamily: "'Press Start 2P', cursive",
+              }}
+            >
+              -&gt;
+            </Typography>
+            <img src="../../../public/images/towerTwoBase-2.png" alt="Tower" />
+            <Typography
+              variant="h5"
+              sx={{
+                margin: "0 80px",
+                color: "#104603",
+                fontFamily: "'Press Start 2P', cursive",
+              }}
+            >
+              -&gt;
+            </Typography>
+            <img src="../../../public/images/towerTwoBase-3.png" alt="Tower" />
+          </Box>
+          <Typography
+            variant="h4"
+            sx={{
+              padding: "0 0 60px 128px",
+              color: "#104603",
+              textAlign: "left",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            Несколько типов врагов:
+          </Typography>
+          {gameDetails.enemies.map((enemy) => (
+            <ItemComponent
+              key={Math.random()}
+              image={enemy.image}
+              text={enemy.text}
+              smallImage={true}
+            />
+          ))}
+          <Typography
+            variant="h4"
+            sx={{
+              color: "#104603",
+              textAlign: "left",
+              fontFamily: "'Press Start 2P', cursive",
+              margin: "40px 0 60px 128px",
+            }}
+          >
+            Как играть:
+          </Typography>
+          <Typography
+            variant="h5"
+            sx={{
+              padding: "0 0 40px",
+              color: "#104603",
+              textAlign: "left",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            После старта игры через некоторое время по карте начнут идти враги.
+            Нужно строить башни, которые будут атаковать врагов и мешать им
+            дойти до конца карты.
+          </Typography>
+          <Typography
+            variant="h5"
+            sx={{
+              padding: "0 0 40px",
+              color: "#104603",
+              textAlign: "left",
+              fontFamily: "'Press Start 2P', cursive",
+            }}
+          >
+            Игра состоит из нескольких волн или раундов. После убийства всех
+            врагов в текущей волне, после небольшой паузы, начинается следующий
+            раунд, в котором больше врагов, у них больше здоровья и они быстрее
+            перемещаются по карте.
+          </Typography>
+          <Typography
+            variant="h5"
+            sx={{
+              padding: "0 0 80px",
```

Comment:
Вообще, такой подход имеетправо на жизнь, но mui во-первых имеет базовое значение пикселей, которым хорошо пользоваться чтобы верстка была пропорциональной, а во-вторых в mui есть сокращения =-)
```suggestion
              pr: 10,
```